source/turtles/turtle_hero.py

```diff
@@ -66,13 +67,14 @@ def move_down(self, pixels):
 
 class TurtleHero(Turtle):
     def __init__(self, type, size_coeff, name, position):
-        self.image_sheet = pygame.image.load('D:\\Users\\Ola\\PycharmProjects\\turtle_game\\source\\turtles\\sv_turtle_sheet.png').convert_alpha()
+        print(__file__)
```

Comment:
I would prefer to not have such unnecessary lines in project
.
.
.
.
.
.
HUE HUE xd